src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color,int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Ball() {
+
+    }
+
+    public enum ColorEnum {
+        RED,
+        WHITE,
+        GREEN,
+        YELLOW,
+        BLUE,
+        BLACK
+    }
+
+    public String toString() {
```

Comment:
you don't need to override method toString(), use method name() https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-for-loop-for-creating-several-objects-of-the-same-class